mr/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -114,8 +114,8 @@ public String catalogName() {
 
   /**
    * The table properties string needed for the CREATE TABLE ... commands,
-   * like "TBLPROPERTIES('iceberg.catalog'='mycatalog')
-   * @return
+   * like TBLPROPERTIES('iceberg.catalog'='mycatalog')
+   * @return the tables properties string, such as TBLPROPERTIES('iceberg.catalog'='mycatalog')
```

Comment:
nit: add `{@code ` around that TBLPROPERTIES

Reply:
done